pages/home.tsx

```diff
@@ -254,7 +253,16 @@ const Home = () => {
           >
             Learn About Us
           </Button>
-          <CardRowContainer>
+          <Box
```

Comment:
Should we change CardRowContainer's implementation to be Box then? Or is it because the card layout are a little different for each page?

Reply:
Both, lol.. I just changed CardRowContainer to have those box properties-- it's a repeated style in the design, so might as well!
It only works with a row of 3 cards, so a row of 4 for example may need something different.